
// TODO: write decorator to catch error to throw httpsError instead
// const useHttpsError = (value: Function, context: { kind: string, name: name }) {

// }